bindata/network/ovn-kubernetes/common/002-rbac-node.yaml

```diff
@@ -156,6 +156,15 @@ rules:
     - get
     - list
     - watch
+{{- if .DNS_NAME_RESOLVER_ENABLE }}
+- apiGroups: ["network.openshift.io"]
+  resources:
+  - dnsnameresolvers
```

Comment:
Where is the `dnsnameresolvers` CRD applied?

Reply:
The CRD will be created when along with the cluster-dns-operator: https://github.com/openshift/cluster-dns-operator/pull/394/files#diff-d5348dedaad45595f0bd40f1b4730932489203aa511b1432951f2bfad8ab8a79